Code/Tools/AssetBundler/source/utils/utils.cpp

```diff
@@ -356,7 +356,7 @@ namespace AssetBundler
             for (const AZ::IO::Path& gemAbsoluteSourcePath : gemInfo.m_absoluteSourcePaths)
             {
                 // We want to check the path before going through the effort of creating the default Seed List file map
-                if (!AzFramework::StringFunc::StartsWith(seedAbsoluteFilePath, gemAbsoluteSourcePath.Native()))
+                if (!AzFramework::StringFunc::StartsWith(seedAbsoluteFilePath, gemAbsoluteSourcePath.LexicallyNormal().Native()))
```

Comment:
You could also use AZ::IO::Path::IsRelativeTo

Reply:
I will try that, but it will also need to normalize the path before the compare.